spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe PostsController do
+
+  let(:valid_attributes) { {title: 'Lorem', body: 'Ipsum'} }
+
+  let(:invalid_attributes) {
+    skip('Add a hash of attributes invalid for your model')
+  }
+
+  let(:valid_session) { {} }
+
+  describe 'GET index' do
+    it 'assigns all posts as @posts' do
+      post = Post.create! valid_attributes
+      get :index, {}, valid_session
+      expect(assigns(:posts)).to eq([post])
+    end
+  end
+
+  describe 'GET show' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.